modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -19,6 +19,7 @@ Before using the {product-title} (ROSA) CLI, `rosa`, to create {hcp-title-first}
 +
 [source,terminal]
 ----
-$ rosa create account-roles --hosted-cp
+$ ACCOUNT_ROLES_PREFIX="${ACCOUNT_ROLES_PREFIX}"
```

Comment:
If you're adding a second command here, it has to be in a separate code block

Reply:
separated